### ISSUE TITLE
Make a contacts app

### ISSUE THREAD

**Issue by gun4qmm7h** [Chooloo/koler#452]
### Checklist

- [x] I am using the latest version - x.xx.x 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Would you be able to make a contacts app, there are no good apps out there and I've falling in love with this UI

#### Is your feature request related to a problem? Please describe it
no

#### How will you/everyone benefit from this feature?

Making a contacts app will be a good idea since there are no good Foss apps out there so you will be getting a lot more attention and people will also see the phone app and maybe you will get more download/support/donations

**Comment by mikenrafter** [comment]
I agree! And it already looks like a lot of the groundwork has been done. This app already has a contacts list, and a popup menu system.

All that would be needed is a contact import/export/edit/display feature. This functionality could be pulled from other open source apps, and slapped into this UI.

Currently I use: [Simple Contacts](https://github.com/SimpleMobileTools/Simple-Contacts), but this app's UI is awful, and it's missing an add to existing contact feature when adding a phone number via intent.

**Comment by anotherus3r** [comment]
This could be done like some other dialer apps, where the dialer and contacts are the same app but have two app shortcuts - one that launches to the dialer tab and one that launches to contacts tab inside the same app.

**Comment by slim35reaper7** [comment]
It'd be so amazing to have a contacts app to make the phone+contacts experience smooth and consistent
